packages/block-library/src/image/block.json

```diff
@@ -68,6 +69,10 @@
 			"source": "attribute",
 			"selector": "figure > a",
 			"attribute": "target"
+		},
+		"isInGallery": {
```

Comment:
Set this for getting initial move to an unordered list sorted, but this probably makes more sense as a `context` so will switch to that tomorrow